icu4c/source/data/cldr-icu-readme.txt

```diff
@@ -226,7 +226,7 @@ make check 2>&1 | tee /tmp/icu4c-newData-makeCheck.txt
 # Repeat steps 4-7 until there are no errors.
 
 # 9. You can also run the make check tests in exhaustive mode (these will also
-# be run automatically on the master branch after the merge resulting from this
+# be run automatically on the primary branch after the merge resulting from this
```

Comment:
It's still the "master branch" until GitHub gives us better tools to change that.

However, what happens in git/GitHub is that they will allow us to change the name of the default branch more easily, and I think we could use that:
`# be run automatically on the default branch after the merge resulting from this`

Reply:
Good suggestion. Done.